src/OmniSharp.Roslyn.CSharp/Services/Signatures/InvocationContext.cs

```diff
@@ -11,7 +12,24 @@ internal class InvocationContext
         public int Position { get; set; }
```

Comment:
Not your fault, but do these properties actually need to be mutable? Can we have them be get-only?

Reply:
Yes , it works with get-only.